src/helpers.js

```diff
@@ -421,7 +421,7 @@ export function createStreamInfo(item, mediaSource, startPosition) {
         let subStreamCodec = subtitleStream.Codec.toLowerCase();
         if (subStreamCodec !== 'vtt' &&
             subStreamCodec !== 'webvtt' &&
-            (subStreamCodec === 'srt' && subtitleStream.DeliveryMethod !== "External")) {
+            !(subStreamCodec === 'srt' && subtitleStream.DeliveryMethod !== "External")) {
```

Comment:
Have you tested this with external srts? I think this change is breaking that

Reply:
Wait no, I'm being stupid here hehe

Reply:
I can invert this logic so it'll be human readable